isis/src/qisis/objs/StretchTool/AdvancedStretchDialog.cpp

```diff
@@ -69,6 +72,7 @@ namespace Isis {
             this, SIGNAL(stretchChanged()));
     connect(p_bluStretch, SIGNAL(stretchChanged()),
             this, SIGNAL(stretchChanged()));
+    // add signal for RGB
```

Comment:
is this a TODO?

Reply:
Yes, but beyond this scope of this week. Maybe I'll make a ticket.